docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
         read_only: true
       - type: bind
         source: ${HOST_SCENARIO_INPUTS_PATH}
-        target: /mnt/scenario-sources
+        target: /mnt/workflow-scenario-preparation-outputs
```

Comment:
this
https://github.com/RMI-PACTA/workflow.data.preparation/blob/8741ed5d6d5e2163cee6f34396362a3763999801/config.yml#L47

will have to be updated to match too

Reply:
Done in [3a362cc](https://github.com/RMI-PACTA/workflow.data.preparation/pull/219/commits/3a362cc4571b316398cbb8b7859979f524493b07)